README.md

```diff
@@ -4,7 +4,12 @@ The main credential are automatically set up for you, howver, the **databaseURL*
 Use the following command to set the **databaseURL** & the **storageBucket** URLs:
   
 ```sh
-$ firebase functions:config:set audioseva.database_url="THE DATABASE URL" audioseva.storage_bucket="THE STORAGE BUCKET URL"
+$ firebase functions:config:set audioseva.database_url="THE DATABASE URL"
+$ firebase functions:config:set audioseva.storage_bucket="THE STORAGE BUCKET URL"
+$ firebase functions:config:set audioseva.send_in_blue.key="sendInBlue secret Key"
+# Coordinator details
+$ firebase functions:config:set audioseva.coordinator.templateid=NUMBER
```

Comment:
Will not work out. There will be many templates for various notifications.
```suggestion
$ firebase functions:config:set audioseva.sqr.allotment.emailTemplateid=NUMBER
```

Reply:
wouldn't it be better if I specify the kind of email recipient ( devotee or coordinator ) ?
**`sqr.allotment.emailTemplateid`** can be somehow ambiguous